### PR TITLE
changelog for 6.0

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -4,11 +4,19 @@ Changes in IPython Parallel
 ===========================
 
 
-5.3
+6.0
 ---
 
+This is a major release due to a compatibility change, but not a big one.
```
Comment (willingc) [inline docs/source/changelog.rst]:
```
Due to a compatibility change and semver, this is a major release. However, it is not a big release.
The main compatibility change is that all timestamps are now timezone-aware...
```

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -4,11 +4,19 @@ Changes in IPython Parallel
...
+The main compatibility change is that timestamps everywhere are now timezone-aware UTC timestamps.
+This means you may see comparison errors if you have code that uses datetime objects without timezone info (so-called naïve datetime objects).
+
+Other fixes:
+
 - Rename :meth:`Client.become_distributed` to :meth:`Client.become_dask`.
   :meth:`become_distributed` remains as an alias.
+- import joblib from a public API for joblib backend
```
Comment (willingc) [inline docs/source/changelog.rst]: Is this change to take advantage of using https://pythonhosted.org/joblib/parallel.html#custom-backend-api-experimental ?

File: docs/source/changelog.rst (hunk 1 above)
Comment (minrk) [inline docs/source/changelog.rst]: Thanks!

File: docs/source/changelog.rst (hunk 2 above)
Comment (minrk) [inline docs/source/changelog.rst]: Yes! It was already, but the first implementation didn't have a public API, yet, so it was a private API import. This has been fixed.

**Comment by willingc** [comment]
Thanks 👍

**Pull request merged**